src/ModuleLoader.ts

```diff
@@ -95,9 +97,16 @@ export class ModuleLoader {
 			: () => true;
 	}
 
-	async addAdditionalModules(unresolvedModules: readonly string[]): Promise<Module[]> {
+	async addAdditionalModules(
+		unresolvedModules: readonly string[],
+		isAddForManualChunks = false
```

Comment:
This fallback is never used and thus messes up coverage

Reply:
removed this fallback